### PR TITLE
Include Support for Selling Ads

### PR DESCRIPTION
**Pull request by Cobra-Bitcoin**
This pull request introduces code that allows Bitcoin.org to show ads which will be located at the top of the site. The ads aren't shown to mobile visitors.

Note: this pull request itself does not contain *any* ads. This just sets up the infrastructure necessary for us to sell ads when we need too.

Technical feedback is appreciated. I'd prefer that we avoid getting into debates about how Bitcoin.org should be funded in this pull request. I don't think that belongs here.

### CODE REVIEW

**Comment by jonasschnelli** [comment]
Related to: https://github.com/bitcoin-dot-org/bitcoin.org/pull/1087

File: ad-auth.txt [hunk 1]
```diff
@@ -0,0 +1,3 @@
+Cøbra (https://bitcointalk.org/index.php?action=profile;u=249495) is authorized
+to sell advertising slots for this website (https://bitcoin.org).
+
```
Comment (luke-jr) [inline ad-auth.txt]: This should not be merged until such authorization has consensus. (Yes, this isn't a technical comment, but neither is this part of the change technical nor merely adding infrastructure.)

Comment (jonasschnelli) [inline ad-auth.txt]: Agree with @luke-jr

**Comment by harding** [comment]
> I'd prefer that we avoid getting into debates about how Bitcoin.org should be funded in this pull request.

Where can people post to try to change your mind?  I stand by my [earlier comment](https://github.com/bitcoin-dot-org/bitcoin.org/pull/1087#issuecomment-147583676) that I think this is ultimately the domain owners decision (as is all content on this site), but that I also think seeing advertisements makes new people less willing to contribute.  By your not having a debate over the decision and trying to persuade people who doubt this is a good decision, it may also make existing contributors less likely to continue contributing.

On the technical side, I will try to find some time to test this today or on Saturday.

**Comment by theymos** [comment]
I feel like putting it at the top is a bit intrusive. Maybe we should try it in the footer first and see if this generates enough ad income.

There should also be some sort of disclaimer. Currently (unless the ad text will say otherwise), it isn't even clear that it's an ad.

**Comment by Cobra-Bitcoin** [comment]
Thanks for the feedback @theymos. I'll work on changing it to show on the footer, and also add a disclaimer.

**Pull request closed without merging**